templates/octavia/config/octavia.conf

```diff
@@ -11,6 +11,9 @@ tls_cipher_prohibit_list=
 default_listener_tls_versions=TLSv1.2,TLSv1.3
 default_pool_tls_versions=TLSv1.2,TLSv1.3
 [database]
+{{ if (index . "DatabaseConnection") }}
+connection = {{ .DatabaseConnection }}
```

Comment:
this is where the database URL comes from now, this file is copied by the kolla stuff when the container first starts from a volume mount